src/mrack/providers/openstack.py

```diff
@@ -104,7 +105,7 @@ async def init(self, image_names=None):
         object_start = datetime.now()
         flavors, images, limits, networks, ips = await asyncio.gather(
             self.load_flavors(),
```

Comment:
From [Azure CI](https://dev.azure.com/neoave/mrack/_build/results?buildId=18&view=logs&j=00b6a206-eb91-585b-3d14-cf1a4d7b1970&t=6de672dc-c37a-5c78-a29e-58cd8097cede&l=70):
```
##[warning]coroutine 'OpenStackProvider.load_flavors' was never awaited
``` 
Is it a regression?

Reply:
I hope not.

Reply:
Seems like it was side effect of failing test. I have pushed the fix for the test already.